src/components/layout.tsx

```diff
@@ -6,15 +6,15 @@
  */
 
 import { graphql, useStaticQuery } from 'gatsby'
-import React from 'react'
+import React, { ReactElement, ReactNode } from 'react'
 
 import { Theme } from '../theme'
 
 interface Props {
-  children?: any
+  children?: ReactNode
 }
 
-const Layout = ({ children }: Props) => {
+const Layout = ({ children }: Props): ReactElement | null => {
```

Comment:
V jakém případe by funkce vracela null? Nějak to v kódu nikde nevidím (to platí i pro další podobné úpravy, kdy se vrací `| null`).